src/deploy/RPM_build/packagerpm.sh

```diff
@@ -33,9 +34,17 @@ function move_builds() {
 }
 
 function get_node_tar() {
+    local arch=$(get_arch)
+    local filename="node-v${nodever}-linux-${arch}.tar.xz"
+
     pushd ~/rpmbuild/SOURCES/
-    local path=$(NODEJS_VERSION=${nodever} download_node)
-    mv ${path} node-${nodever}.tar.xz
+    if [ -f ${NODE_PATH}/${filename} ]
+    then
+        mv ${NODE_PATH}/${filename} node-${nodever}.tar.xz
```

Comment:
Don't we have to tar it again here because we untarred it on `install_nodejs.sh:54`? Also, I think we should `cp` instead of `mv` because we symlinked previously on `install_nodejs.sh`.

Reply:
the tar in `install_nodejs.sh:54` is not removing the tarball, just opening it, so we have the tared file. 
the symlink is a link of the open files, so we can move the tar.
also previously we rm the file, and now we are not, as we don't need it clean on the base image and we can move it wherever we want.